src/main/kotlin/it/pagopa/wallet/scheduler/services/WalletService.kt

```diff
@@ -0,0 +1,46 @@
+package it.pagopa.wallet.scheduler.services
+
+import it.pagopa.wallet.scheduler.config.WalletSearchConfig
+import it.pagopa.wallet.scheduler.documents.Wallet
+import it.pagopa.wallet.scheduler.exceptions.WalletInvalidRangeException
+import it.pagopa.wallet.scheduler.repositories.WalletRepository
+import java.time.Instant
+import java.util.*
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import reactor.core.publisher.Mono
+
+@Service
+class WalletService(
+    @Autowired val walletRepository: WalletRepository,
+    @Autowired val walletSearchConfig: WalletSearchConfig
+) {
+    val logger: Logger = LoggerFactory.getLogger(this.javaClass)
+
+    fun getWalletsForCdcIngestion(startDate: Instant, endDate: Instant): Mono<List<Wallet>> {
```

Comment:
Please consider changing the return type to Flux<Wallet>, matching the return type of WalletRepository. At the moment we are forcing the client to create a Flux from an iterable #6

Reply:
done here 11e7bc5bdfd7c0e33a69e907262b46696d46cfc6